### PR TITLE
Kata 10: baseline relevance case

### PR DESCRIPTION
**Pull request by epugh**

### CODE REVIEW

**Review by david-fisher — COMMENTED** [review]
Suggesting binary relevance (line 15) is inconsistent with then suggesting nDCG@10, which generally uses graded relevance.

File: quickstart.sh [hunk 1]
```diff
@@ -189,9 +189,10 @@ if $offline_lab; then
   echo -e "${MAJOR}Setting up Quepid${RESET}"
   docker-compose run --rm quepid bin/rake db:setup
   docker-compose run quepid thor user:create -a admin@choruselectronics.com "Chorus Admin" password
+  echo -e "${MINOR}Setting up Chorus Baseline Relevance case${RESET}"
   docker-compose run quepid thor case:create "Chorus Baseline Relevance" solr http://localhost:8983/solr/ecommerce/select JSONP "id:id, title:title, thumb:img_500x500, name, brand, product_type" "q=#\$query##&useParams=visible_products,querqy_algo" nDCG@10 admin@choruselectronics.com
   docker cp ./katas/Broad_Query_Set_rated.csv quepid:/srv/app/Broad_Query_Set_rated.csv
-  docker exec quepid thor ratings:import 1 /srv/app/Broad_Query_Set_rated.csv
+  docker exec quepid thor ratings:import 1 /srv/app/Broad_Query_Set_rated.csv >> /dev/null
```
Comment (risdenk) [inline quickstart.sh:195]: why `>> /dev/null`? what errors does it show? Could it show errors that would be useful?

File: katas/010_establishing_a_baseline_relevance.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+We see this effect daily in our relevancy tuning work.  You improve the results for one query, and that change has unanticipated impacts on many other queries.   Some times the unanticipated impacts are a pleasant surprise, improving many other queries.  More commonly, it has a negative impact on other queries.    
+
+To deal with this, to use another analogy; _we need to look at the forest, not the trees_, and we do this by creating a Baseline Relevancy Case in Quepid.
+
+The Baseline Relevancy Case is a set of queries that represents the typical queries we are getting.   You should source them from your query logs to they represent real queries, and it should have queries that come from both your Head and your Long Tail of queries that users are running.  100 queries is typically enough to get started, though more is always better ;-).
```
Comment (wrigleyDan) [inline katas/010_establishing_a_baseline_relevance.md:13]:
```suggestion
The Baseline Relevancy Case is a set of queries that represents the typical queries we are getting. You should source them from your query logs so they represent real queries, and it should have queries that come from both your Head and your Long Tail of queries that users are running. 100 queries is typically enough to get started, though more is always better ;-).
```

**Review by wrigleyDan — CHANGES_REQUESTED** [review]

**Comment by epugh** [comment]
Not worth delaying this kata, but we did find https://github.com/o19s/quepid/issues/581

File: quickstart.sh (hunk 1 above)
Comment (epugh) [reply quickstart.sh:195]: You actually see a lot noise from the command....    Not errors, but just ltos of extra loggign that overwhelms the log output..

**Pull request merged**